CONTRIBUTING.md

```diff
@@ -94,6 +98,13 @@ conda activate shap
 
 ### Installing from source
 
+To build from source, you need a compiler to build the C extension. For example
+on linux, you can install gcc with:
+
+```bash
+sudo apt install build-essential
+```
+
```

Comment:
On Windows, there are multiple ways, one is to install mingw64 from [here](https://www.mingw-w64.org/downloads/)